include/api.php

```diff
@@ -4078,26 +4078,18 @@ function api_fr_photoalbum_delete($type)
 		throw new BadRequestException("no albumname specified");
 	}
 	// check if album is existing
-	$r = q(
-		"SELECT DISTINCT `resource-id` FROM `photo` WHERE `uid` = %d AND `album` = '%s'",
-		intval(api_user()),
-		DBA::escape($album)
-	);
-	if (!DBA::isResult($r)) {
+
+	$photos = DBA::selectToArray('photo', ['resource-id'], ['uid' => api_user(), 'album' => $album]);
```

Comment:
Could you use the `group_by` parameter in the database call? See for example here: https://github.com/friendica/friendica/blob/develop/include/api.php#L1559

Then we don't need to call this `array_unique`

Reply:
Yup, good point.

Reply:
Done.